scripts/leaderboard.coffee

```diff
@@ -206,33 +206,43 @@ module.exports = (robot) ->
     msg.send response
 
   # response for adding to leaderboard
-  robot.respond /debut ([\w\-_]+)/i, (msg) ->
-    name = msg.match[1]
-    ScoreField = scorefield()
-    Aliases = aliases()
-    if verifyName(name, ScoreField, Aliases)
-      response = "#{name} is already in the game."
-      msg.send response
-    else
-      ScoreField[name.toLowerCase()] = 0
-      response = "Added #{name} to roster."
-      msg.send response
+  robot.respond /debut (.+)/i, (msg) ->
+    debut = msg.match[1].toLowerCase()
+    queries = debut.split " "
+    name_verified=""
+    name_unset=""
+    for name in queries
+      ScoreField = scorefield()
+      Aliases = aliases()
+      if verifyName(name, ScoreField, Aliases)
+        name_verified=name_verified.concat(" ").concat name
+      else
+        ScoreField[name.toLowerCase()] = 0
+        name_unset=name_unset.concat name.concat(" ")
+    if name_verified!="" 
+      msg.send "#{name_verified}is(are) already in the game."
```

Comment:
add space before is.

Reply:
also have spaces between and after equal operator like it is present in the rest of the file. Consistency improves readability.